src/__main__.py

```diff
@@ -77,7 +85,7 @@ def get_total_lines_of_code(report_code_lines):
         re.findall(REGEX_TO_MATCH_WITH_ROW_TOTALS, report_code_lines)[0]
     )
     total_lines, blank_lines = convert_number_string_to_number_list(
-        total_lines_row
+        total_lines_row, ","
```

Comment:
En estos casos, por fines de documentación puede ser interesante indicar qué parámetro estamos seteando, para no tener que ver la definición de la función para entender qué está pasando.
```suggestion
        total_lines_row, separator=","
```

Pero lo dejo a tu criterio.